app/presenters/idv/welcome_presenter.rb

```diff
@@ -73,10 +74,15 @@ def bullet_points
 
     private
 
-    attr_accessor :decorated_sp_session
+    attr_accessor :decorated_sp_session, :current_user
 
     def bullet_point(bullet, text)
       OpenStruct.new(bullet: bullet, text: text)
     end
+
+    def stepping_up?
```

Comment:
Nit: Can we use a more descriptive name than `stepping_up?` I liked having the reference to `selfie` in there.

Reply:
Changed